coss/home/models.py

```diff
@@ -20,36 +22,80 @@ class Meta:
 class AboutSnippet(models.Model):
     title = models.CharField(default='', max_length=40)
     header_text = fields.RichTextField(blank=True)
-    body_title = models.CharField(blank=True, default='', max_length=40)
-    body_text = fields.RichTextField(blank=True)
-    info_title = models.CharField(blank=True, default='', max_length=40)
-    info_text = fields.RichTextField(blank=True)
-    bottom_cta_title = models.CharField('Title', blank=True, default='', max_length=40)
-    bottom_cta_text = models.CharField('Text', blank=True, default='', max_length=40)
-    bottom_cta1_text = models.CharField('Text', blank=True, default='', max_length=20)
-    bottom_cta1_link = models.URLField('Link', blank=True, default='')
-    bottom_cta2_text = models.CharField('Text', blank=True, default='', max_length=20)
-    bottom_cta2_link = models.URLField('Link', blank=True, default='')
+    body_title = models.CharField(verbose_name='Title', blank=True, default='',
+                                  max_length=40)
+    body_text = fields.RichTextField(verbose_name='Text', blank=True)
+    body_image = models.ForeignKey(
+        'wagtailimages.Image',
```

Comment:
Locally, the image that I chose is not displayed in the template. Please make sure that this is working properly before merging this PR. Also that the fallback is working too.